app/bundles/LeadBundle/Form/Type/ContactFrequencyType.php

```diff
@@ -192,12 +192,12 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'lead_lists',
                 'leadlist_choices',
                 [
-                    'global_only' => true,
```

Comment:
Is there a reason that this line is removed?

Reply:
Because we have new option `preference_center_only `
global_only will make sense If we will use in future permission for segmetns (same like in reports - public report)
https://github.com/mautic/mautic/blob/4c8b45b548ae08fac5f44119f7806c41b79de988/app/bundles/LeadBundle/Form/Type/ContactFrequencyType.php#L195